lib/Config.cpp

```diff
@@ -139,13 +163,16 @@ LibConfig::ReturnedValue LibConfig::Config::getReadOnlyKey(Key *configKey) const
 }
 
 ///
-/// \todo implementation
+/// \todo handle error
 ///
 LibConfig::ReturnedValue LibConfig::Config::getSettingValue(char const *settingName, char *value, size_t valueSize) const
 {
-  (void)settingName;
-  (void)value;
-  (void)valueSize;
+  json request;
+  request["REQUEST_NAME"] = "SETTING_GET";
+  request["CONFIG_ID"] = configId;
+  request["SETTING_NAME"] = settingName;
+  sendJson(request);
+  std::memcpy(value, lastRequestedValue.c_str(), (valueSize < lastRequestedValue.length() ? valueSize : lastRequestedValue.length()));
```

Comment:
```suggestion
  std::memcpy(value, lastRequestedValue.c_str(), std::min(valueSize, lastRequestedValue.length()));
```
(might need an additional include to compile)